README.md

```diff
@@ -31,48 +33,66 @@ Restate supports the following SDKs:
 * [Typescript](https://github.com/restatedev/sdk-typescript)
 * [Java and Kotlin](https://github.com/restatedev/sdk-java)
 
-## Building Restate
+## Install
 
-In order to build Restate locally [follow the build instructions](https://github.com/restatedev/restate/blob/main/docs/dev/local-development.md#building-restate).
+We offer pre-built binaries of the CLI and the server for MacOS and Linux.
 
-## Running Restate
+### Install the server
```

Comment:
We're missing the docker image, i feel this is something we should have somewhere in the readme, perhaps somewhere here in this paragraph, like:

Pull the container image:

```
docker pull docker.io/restatedev/restate
```

Reply:
I've pushed an fixup adding how to run the Restate server via `docker`.